troposphere/static/js/components/modals/image_version/ImageSelect.react.js

```diff
@@ -11,7 +11,12 @@ export default React.createClass({
     },
 
     handleChange: function(e){
-      this.props.onChange(e.target.value)
+      let image_id = e.target.value;
+      let username = stores.ProfileStore.get().get('username'),
+          image = stores.ImageStore.fetchWhere({
+          created_by__username: username
+      }).get(image_id);
```

Comment:
same call is made here: https://github.com/iPlantCollaborativeOpenSource/troposphere/pull/399/files#diff-a75bff3a62a97aecc360c4adef0e4e37R35